src/line/line.html

```diff
@@ -32,7 +32,7 @@ <h1>Line Chart Builder</h1>
                     </div>
                     <div class="content">
                         <div class="data-entry">
-                            <div id="x-y-data-grid">
+                            <div id="x-y-data-grid" data-testid = "coordvalues">
```

Comment:
I might be blind, but I couldn't find where you were using the data-testid?

Reply:
I compleltly forgot I had added that. I found a way to reference that element without the test id. The data-testid = "coordvalues" can be removed. It isn't nessesary.